recipes/mycotools/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "mycotools" %}
+{% set version = "0.31.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mycotools-{{ version }}.tar.gz
+  sha256: 14a9abd29c6ed066d9df04b7461b19bb8b45922808de8a2efd03e1d40dab61b1
+
+build:
+  entry_points:
+    - mtdb = mycotools.mtdb:main
+    - acc2fa = mycotools.acc2fa:cli
+    - acc2gbk = mycotools.acc2gbk:cli
+    - acc2gff = mycotools.acc2gff:cli
+    - acc2locus = mycotools.acc2locus:cli
+    - add2gff = mycotools.add2gff:cli
+    - annotationStats = mycotools.annotationStats:cli
+    - assemblyStats = mycotools.assemblyStats:cli
+    - bioreform = mycotools.bioreform:cli
+    - coords2fa = mycotools.coords2fa:cli
+    - crap = mycotools.crap:cli
+    - db2files = mycotools.db2files:cli
+    - db2hgs = mycotools.db2hgs:cli
+    - db2microsyntree = mycotools.db2microsyntree:cli
+    - db2search = mycotools.db2search:cli
+    - extract_mtdb = mycotools.extract_mtdb:cli
+    - fa2clus = mycotools.fa2clus:cli
+    - fa2hmmer2fa = mycotools.fa2hmmer2fa:cli
+    - fa2mass = mycotools.fa2mass:cli
+    - fa2tree = mycotools.fa2tree:cli
+    - fna2faa = mycotools.fna2faa:cli
+    - gff2seq = mycotools.gff2seq:cli
+    - gff2svg = mycotools.gff2svg:cli
+    - jgiDwnld = mycotools.jgiDwnld:cli
+    - manage_mtdb = mycotools.manage_mtdb:cli
+    - ncbiAcc2fa = mycotools.ncbiAcc2fa:cli
+    - ncbiDwnld = mycotools.ncbiDwnld:cli
+    - ome2name = mycotools.ome2name:cli
+    - predb2mtdb = mycotools.predb2mtdb:cli
+    - s2subs = mycotools.s2subs:cli
+    - update_mtdb = mycotools.update_mtdb:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mycotools', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.0,<4.0
+    - setuptools
```

Comment:
Those pins are not needed. Please remove them.

Reply:
This problem has been addressed and the checks are passed now that the Azure pipeline is running again.